tests/Behat/Gherkin/Cache/FileCacheTest.php

```diff
@@ -48,15 +48,15 @@ public function testCacheAndRead()
 
     public function testBrokenCacheRead()
     {
-        $this->setExpectedException('Behat\Gherkin\Exception\CacheException');
+        $this->expectException('Behat\Gherkin\Exception\CacheException');
 
         touch($this->path . '/v' . Gherkin::VERSION . '/' . md5('broken_feature') . '.feature.cache');
         $this->cache->read('broken_feature');
     }
 
     public function testUnwriteableCacheDir()
     {
-        $this->setExpectedException('Behat\Gherkin\Exception\CacheException');
+        $this->expectException('Behat\Gherkin\Exception\CacheException');
```

Comment:
I think this method does not exist in 4.8

Reply:
I adjusted `composer.json` to officially support only PHP 5.6 upwards and remove mention of PHPUnit 4.

Reply:
And in some later PR we can easily move all the way up to just PHP 7.2+ and the much newer PHPUnit versions.